src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2476,6 +2479,18 @@ public void deleteTranslation(Long translationId) {
                 handleStatus()));
     }
 
+    public ArticleAttachments createArticleAttachment(Long articleId, boolean inline, String guideMediaId, String locale) {
+        Map<String, Object> request = new HashMap<>();
+        request.put("inline", inline);
+        request.put("guide_media_id", guideMediaId);
+        request.put("locale", locale);
+
+        TemplateUri tmpl = tmpl("/help_center/articles/{articleId}/attachments")
```

Comment:
Url for creating attachment is `https://{{subdomain}}.zendesk.com/api/v2/help_center/{{targetLocale}}/articles/{{articleId}}/attachments`. locale in URL but not in the form params.

Reply:
I believe it's important and I prefer Pavel's URL. There are 2 reasons:
* Their documentation mentions that an attachment will be created for a specific locale. Maybe there is no difference in API behavior now, but it can be changed in the future. Pavel's URL explicitly includes locale, so it's much easier to argue that we do everything correctly.
* There is a similar problem with Article translations (two API endpoints). But when you have to deal with ZD support, they are picky about every little thing, so it would be good to use the correct URL.

Reply:
If you insist, I will change.
But it isn't a good way to use an undocumented endpoint.